ee/dtr/admin/configure/deploy-caches/simple-kube.md

```diff
@@ -0,0 +1,229 @@
+        options:
+          blobttl: 24h
+          upstreams:
+            - https://<dtr-url>
+          cas:
+            - /certs/dtr.cert.pem
+```
+
+With this configuration, the cache fetches image layers from DTR and keeps
+a local copy for 24 hours. After that, if a user requests that image layer,
+the cache fetches it again from DTR.
+
+The cache is configured to persist data inside its container.
+If something goes wrong with the cache service, Docker automatically redeploys a
+new container, but previously cached data is not persisted.
+You can [customize the storage parameters](/registry/configuration.md#storage),
+if you want to store the image layers using a persistent storage backend.
+
+### Create Kubernetes Resources
+
+Create a Kubernetes namespace to logical seperate all of our DTR cache components.
+
+```
+$ kubectl create ns dtr
+```
+
+Create the Kubernetes Secrets and the Kubernetes ConfigMaps. Note the commands below
+will only work if your workstation directory was configured our examples structure 
+above. 
+
+```
+$ kubectl create secret generic dtr-certs \
+    --from-file=certs/dtr.cert.pem \
+    --from-file=certs/cache.cert.pem \
+    --from-file=certs/cache.key.pem
+
+$ kubectl create configmap dtr-cache-config \
+    --from-file=config.yml
+```
+
+Create the Kubernetes Deployment. 
+
+```
+cat <<EOF | kubectl create -f -
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: dtr-cache
+  namespace: dtr
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: dtr-cache
+  template:
+    metadata:
+      labels:
+        app: dtr-cache
+      annotations:
+       seccomp.security.alpha.kubernetes.io/pod: docker/default
+    spec:
+      containers:
+        - name: dtr-cache
+          image: {{ page.dtr_org }}/{{ page.dtr_repo }}-content-cache:{{ page.dtr_version }}
+          command: ["bin/sh"]
+          args:
+            - start.sh
+            - /config/config.yml
+          ports:
+          - name: https
+            containerPort: 443
+          volumeMounts:
+          - name: dtr-certs
+            readOnly: true
+            mountPath: /certs/
+          - name: dtr-cache-config
+            readOnly: true
+            mountPath: /config
+      volumes:
+      - name: dtr-certs
+        secret:
+          secretName: dtr-certs
+      - name: dtr-cache-config
+        configMap:
+          defaultMode: 0666
+          name: dtr-cache-config
+EOF
+```
+
+You can check if the deployment has been successful with `kubectl get pods -n dtr`.
+If you need to troubleshoot your deployment, you can use `kubectl describe -n dtr <pods>`
+and / or `kubectl logs -n dtr <pods>`
+
+For external access to the DTR cache you could expose your DTR cache through 
+multiple interfaces: Ingress Object, Node Port, Host Port or Loadbalancer. 
+The following yaml will expose the DTR cache through an ingress obect.
+
+> Note an ingress controller is a pre-requsite for this example. If you have not deploy
```

Comment:
It took analyzing the existing screenshot to infer that `curl I <ucp-url>:<ingressport>` really meant the UCP URL, since the URL doesn't seem to reflect on the current interface like it does here: https://docs.docker.com/ee/ucp/kubernetes/layer-7-routing/#check-your-deployment

Reply:
As mentioned above about the ingress doc :)